README.md

```diff
@@ -158,28 +158,20 @@ In this case, you can either try to hunt down the issue yourself or try to work
 ### Manual Download Timeout Workaround
 
 You can work around the launcher's timeout logic by downloading the Terasology artifact manually.
-If you have Java installed and set up, you can start Terasology directly from the artifact (scenario 1 below).
-Otherwise, you'll need to copy the artifact into the launcher directory (scenario 2 below).
-If scenario 1 should not work, you might have an incompatible version or flavor of Java installed or an incompatible setup.
-In this case, please go for scenario 2.
 
 Workaround steps:
 1. Download the desired Terasology artifact from GitHub: https://github.com/MovingBlocks/Terasology/releases/download/v<version>/TerasologyOmega.zip
 2. Extract the artifact (zip archive)
-
-Scenario 1: You have Java installed and set up
-3. Start Terasology<br>
-   On Windows: Execute `Terasology.bat`<br>
-   On Linux: Execute `Terasology`
-
-Scenario 2: You don't have Java installed or set up
-3. Create a new path in the `games` sub-directory of the TerasologyLauncher download directory<br>
-   Path for stable releases: `<games-subdir>/OMEGA/STABLE/<version>`<br>
-   Path for pre-releases: `<games-subdir>/OMEGA/NIGHTLY/<version>`
-4. Copy the contents of the previously downloaded and extracted Terasology artifact into the newly created version directory at the end of the path
-5. Start the launcher and select the desired version, the play icon (:arrow_forward:) should now be available for you to start Terasology
-
-![Download Timeout Workaround](docs/images/221220_TerasologyLauncher_download-timeout-workaround.png)
+3. Rename the archive<br/>
+   For stable releases to `terasology-omega-<version>-stable.zip`, e.g. `terasology-omega-5.3.0-stable.zip`<br/>
```

Comment:
I recently learned that many people actually don't know what "e.g." and "i.e." and so on are, since then I'm mostly using "for example" and "that is". I'm fine with merging, but it might happen that I'll change that at some point :wink: